cypress/support/Helper/Candidates/api-helper.ts

```diff
@@ -0,0 +1,128 @@
+import { candidateData, employeeData, jobData, sheduleInterviewData, userData, vacancyData } from "./payload-function";
+
+const baseUrl = Cypress.config().baseUrl;
+export const URLs: any = {
+  employee: `${baseUrl}/api/v2/pim/employees`,
+  user: `${baseUrl}/api/v2/admin/users`,
+  candidates: `${baseUrl}/api/v2/recruitment/candidates`,
+  vacancy: `${baseUrl}/api/v2/recruitment/vacancies`,
+  job: `${baseUrl}/api/v2/admin/job-titles`,
+};
+
+export const addEmployee = (firstName: string, id: string, lastName: string) => {
```

Comment:
for creating data, personally, I see it more readable to name the function by appending the word `create` instead of `add`, like you can say `createNewEmployee`, ... etc